toolkit/tools/graphpkgfetcher/graphpkgfetcher.go

```diff
@@ -418,7 +418,7 @@ func downloadSingleDeltaRPM(realDependencyGraph *pkggraph.PkgGraph, buildNode *p
 	// already have it in the cache.
 	if !foundCacheRPM {
 		// Avoid any processing since we know the exact RPM we want to download
-		_, err = cloner.CloneRawPackageNames(downloadDependencies, fullyQualifiedRpmName)
+		_, err = cloner.CloneRawPackageNames(downloadDependencies, false /*no transaction*/, fullyQualifiedRpmName)
```

Comment:
Could we use the well-named constants approach we have elsewhere instead of adding a new approach of in-line comments? I'd prefer we stick to one convention.